lib/bunyan.js

```diff
@@ -995,10 +995,16 @@ Logger.stdSerializers.res = function res(res) {
 function getFullErrorStack(ex)
 {
     var ret = ex.stack || ex.toString();
-    if (ex.cause && typeof (ex.cause) === 'function') {
-        var cex = ex.cause();
+    if (ex.cause) {
+        var cex;
+        if (typeof (ex.cause) === 'function') {
+            cex = ex.cause();
+        }
+        if (typeof (ex.cause) === 'object') {
+            cex = ex.cause;
+        }
         if (cex) {
-            ret += '\nCaused by: ' + getFullErrorStack(cex);
+            ret += '\ncaused by ' + getFullErrorStack(cex);
         }
```

Comment:
The default [V8 stack trace formatter](https://code.google.com/p/v8-wiki/wiki/JavaScriptStackTraceApi), that's what I'm talking about.
The error `name` usually starts with a capital and is followed by `:`, thus my small change on the connector string (which should have a low profile).